packages/nimble-components/src/label-provider/rich-text/index.ts

```diff
@@ -5,7 +5,8 @@ import {
     richTextToggleBoldLabel,
     richTextToggleItalicsLabel,
```

Comment:
It's a bit surprising to me how it looks with only a few matches:
![image](https://github.com/ni/nimble/assets/1588923/f4a8ef22-d163-477e-adfe-34ea5bf42f77)

And with no matches:
![image](https://github.com/ni/nimble/assets/1588923/65360ab5-61a4-4278-846b-ea9674a14754)

The combox behavior isn't perfect but is it possible to get closer to that?

Fits to the size when few items
![image](https://github.com/ni/nimble/assets/1588923/6790c5ba-bdc2-4064-b26d-44c206b50e8b)

Gets narrow for no items
![image](https://github.com/ni/nimble/assets/1588923/5a6a93bb-e661-4c98-807e-1df450ba5b37)

Reply:
Updated the styling to match the combobox.